source/textmate_tools.rb

```diff
@@ -2,6 +2,7 @@
 require 'yaml'
 require 'set'
 require 'deep_clone' # gem install deep_clone
+require 'pathname'
```

Comment:
There is no associated Gemfile change

Reply:
I think this is a built-in library but I could be wrong, I'll check it